client/src/components/NavBar/NavBar.tsx

```diff
@@ -3,30 +3,63 @@ import AppBar from '@material-ui/core/AppBar';
 import ToolBar from '@material-ui/core/Toolbar';
 import Button from '@material-ui/core/Button';
 import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
 import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
 import logo from '../../Images/logo.png';
+import { User } from '../../interface/User';
+import AuthMenu from '../AuthMenu/AuthMenu';
 
-export default function NavBar(): JSX.Element {
+interface Props {
+  loggedInUser?: User;
+  handleDrawerToggle?: () => void;
+}
+
+const NavBar = ({ loggedInUser }: Props): JSX.Element => {
   const classes = useStyles();
 
   return (
-    <AppBar position="static">
+    <AppBar className={classes.appbar} position="sticky">
       <ToolBar className={classes.toolbar}>
         <img src={logo} alt="logo" />
-        <Grid className={classes.loginSignup}>
-          <Box p={1}>
-            <Button component={Link} to="/login" color="primary" size="large" variant="outlined">
-              Login
-            </Button>
-          </Box>
-          <Box p={1}>
-            <Button component={Link} to="/signup" color="primary" size="large" variant="contained">
-              Sign Up
-            </Button>
-          </Box>
-        </Grid>
+        {loggedInUser ? (
+          <Grid className={classes.navButtons}>
```

Comment:
Grid items have two types of layouts: containers and items, selecting the 'container' layout here seems appropriate